test/setup/build.ts

```diff
@@ -21,17 +21,22 @@ async function build() {
   rmSync(tmpTestDir, { recursive: true, force: true });
   mkdirSync(tmpTestDir);
 
-  async function downloadFile(path, destinationPath) {
+  async function downloadFile(path: string, destinationPath: string): Promise<void> {
     const done = promisify(stream.finished);
     const writer = createWriteStream(destinationPath);
 
-    await axios({ method: 'get', url: path, responseType: 'stream' }).then(function (response) {
-      response.data.pipe(writer);
-      return done(writer);
-    });
+    await axios({ method: 'get', url: path, responseType: 'stream' })
+      .then(function (response) {
+        response.data.pipe(writer);
+        return done(writer);
+      })
+      .catch((error) => console.log(error));
+  }
+  try {
+    await downloadFile(rustProgramsRepository, `${tmpTestDir}/master.zip`);
+  } catch (error) {
+    console.log(error);
```

Comment:
Ditto reqarding error logging, but you'd never even get here since you already log and then ignore the error inside the `download` method.

Please chose _one_ place to handle the error and then don't just log it but think about what the consequence should be and as I explained above you should probably exit with an error code.